ironfish/src/rpc/routes/transactions/sendTransaction.ts

```diff
@@ -91,6 +93,8 @@ router.register<typeof SendTransactionRequestSchema, SendTransactionResponse>(
       BigInt(transaction.transactionFee),
       transaction.memo,
       transaction.toPublicKey,
+      node.config.get('defaultTransactionExpirationSequenceDelta'),
```

Comment:
This is the only oddity, but I'm going to approve it. I think taking both may make this API a little bit confusing to outsiders.